package.json

```diff
@@ -65,7 +65,7 @@
     "babel-plugin-lodash": "^3.3.4",
     "clean-webpack-plugin": "^0.1.19",
     "connect-history-api-fallback": "^1.5.0",
-    "copy-webpack-plugin": "^4.5.1",
+    "copy-webpack-plugin": "^5.1.1",
```

Comment:
This plugin is probably useless, as it just copies the old Red Hat logo. :smile_cat: